src/components/Item/VisualizationItem/Visualization/plugin.js

```diff
@@ -22,7 +23,9 @@ const itemTypeToScriptPath = {
 }
 
 const hasIntegratedPlugin = (type) =>
-    [CHART, REPORT_TABLE, VISUALIZATION, MAP].includes(type)
+    [CHART, REPORT_TABLE, VISUALIZATION, MAP, EVENT_VISUALIZATION].includes(
```

Comment:
Isn't this only the case (EVENT_VISUALIZATION) for 42?

Reply:
That's a good observation.
I don't think it matters for 42 because `api/apps` returns bundled apps too, so `hasIntegratedPlugin` is actually skipped in 42 for all bundled apps.
We should then revert this code so it works as it does currently in 40/41.